packages/nx/src/executors/run-commands/run-commands.impl.ts

```diff
@@ -312,7 +312,15 @@ function parseArgs(options: RunCommandsOptions) {
     const unknownOptionsTreatedAsArgs = Object.keys(options)
       .filter((p) => propKeys.indexOf(p) === -1)
       .reduce((m, c) => ((m[c] = options[c]), m), {});
-    return unknownOptionsTreatedAsArgs;
+
+    const unparsedCommandArgs = yargsParser(options.__unparsed__, {
```

Comment:
Can we please add some unit tests here to make sure that things are working as expected?

Reply:
can do, a unit test actually failed after the change so there is at least one scenario covered